matlab_proxy/matlab_proxy/__init__.py

```diff
@@ -108,6 +108,20 @@ def invoke(self, name, args, nargout, bare, input_names, output_names):
         out = six.StringIO()
         err = six.StringIO()
         args = pickle.loads(b64decode(args))
+
+        def convert_arg_to_matlab_datatype(arg):
+            if isinstance(arg, list):
+                if len(arg) and isinstance(arg[0], six.string_types):
+                    return arg
+                else:
+                    return matlab.double(arg)
+            elif isinstance(arg, dict):
+                for k, v in arg.iteritems():

```

Comment:
This is not Python 3 compatible

Reply:
https://six.readthedocs.io/#six.iteritems

Reply:
Corrected
https://github.com/metamorph-inc/matlab_wrapper/commit/d207a403625c5c6734f662466878febb50795273